src/views/SpaceDelegates.vue

```diff
@@ -1,6 +1,7 @@
 <script setup lang="ts">
 import { ExtendedSpace } from '@/helpers/interfaces';
 import { useInfiniteScroll, refDebounced } from '@vueuse/core';
+import { DelegationTypes } from '../helpers/delegation/index';
```

Comment:
```suggestion
import { DelegationTypes } from '../helpers/delegation';
```

Reply:
Please remove index everywhere as it will default to index, and use absolute paths `@` instead of relative paths